sample-apps/payment-customizations/extensions/payment-customization-js/src/run.graphql

```diff
@@ -1,4 +1,4 @@
-query Input {
```

Comment:
Nit: Is there a reason why this name isn't consistent across javascript and rust?

Reply:
Code generation requirements 😿 ... we'd eventually like to align Rust codegen with what we do for JS/TS to resolve this.